### PR TITLE
Fix/placing

### PR DESCRIPTION
**Pull request by jlunenburg**

### CODE REVIEW

**Comment by LoyVanBeek** [comment]
@MatthijsBurgh the code looks good but no time to test anymore today. 
Can you test placing at different heights? (e.g. in the bookcase)

File: robot_smach_states/src/robot_smach_states/manipulation/place.py [hunk 1]
```diff
@@ -119,24 +121,29 @@ def execute(self, userdata):
...
-        if not arm.send_goal(place_pose_bl, timeout=10, pre_grasp=False):
+        if not arm.send_goal(kdlFrameStampedFromXYZRPY(place_pose_bl.frame.p.x(),
+                                                       place_pose_bl.frame.p.y(),
+                                                       height+0.15, 0.0, 0.0, 0.0,
+                                                       frame_id="/{0}/base_link".format(self._robot.robot_name)),
+                                                       timeout=10,
+                                                       pre_grasp=True):
             # If we can't place, try a little closer
-            place_pose_bl.frame.p.x(place_pose_bl.frame.p.x() - 0.05)
+            place_pose_bl.frame.p.x(place_pose_bl.frame.p.x() - 0.025)
```
Comment (reinzor) [inline robot_smach_states/src/robot_smach_states/manipulation/place.py]: Hardcoded number ...

File: robot_smach_states/src/robot_smach_states/manipulation/place.py [hunk 2]
```diff
@@ -119,24 +121,29 @@ def execute(self, userdata):
...
 
             rospy.loginfo("Retrying preplace")
             if not arm.send_goal(kdlFrameStampedFromXYZRPY(place_pose_bl.frame.p.x(),
                                                            place_pose_bl.frame.p.y(),
-                                                           height+0.2, 0.0, 0.0, 0.0,
+                                                           height+0.15, 0.0, 0.0, 0.0,
```
Comment (reinzor) [inline robot_smach_states/src/robot_smach_states/manipulation/place.py]: Hardcoded number ...

File: robot_smach_states/src/robot_smach_states/manipulation/place.py [hunk 3]
```diff
@@ -119,24 +121,29 @@ def execute(self, userdata):
...
+                                 timeout=10, pre_grasp=True):
                 rospy.logwarn("Cannot pre-place the object")
                 arm.cancel_goals()
                 return 'failed'
 
         # Place
         if not arm.send_goal(kdlFrameStampedFromXYZRPY(place_pose_bl.frame.p.x(),
                                                        place_pose_bl.frame.p.y(),
-                                                       height+0.15, 0.0, 0.0, 0.0,
+                                                       height+0.1, 0.0, 0.0, 0.0,
```
Comment (reinzor) [inline robot_smach_states/src/robot_smach_states/manipulation/place.py]: Hardcoded number ...

File: robot_smach_states/src/robot_smach_states/manipulation/place.py [hunk 4]
```diff
@@ -157,7 +164,7 @@ def execute(self, userdata):
         # Retract
         arm.send_goal(kdlFrameStampedFromXYZRPY(place_pose_bl.frame.p.x() - 0.1,
                                                 place_pose_bl.frame.p.y(),
-                                                place_pose_bl.frame.p.z() + 0.05, 0.0, 0.0, 0.0,
+                                                place_pose_bl.frame.p.z() + 0.15, 0.0, 0.0, 0.0,
```
Comment (reinzor) [inline robot_smach_states/src/robot_smach_states/manipulation/place.py]: Hardcoded number ...

File: robot_smach_states/src/robot_smach_states/util/designators/ed_designators.py [hunk 5]
```diff
@@ -300,7 +300,7 @@ def __init__(self, robot, place_location_designator, name=None, area=None):
         self.robot = robot
 
         self.place_location_designator = place_location_designator
-        self._edge_distance = 0.1  # Distance to table edge
+        self._edge_distance = 0.05  # Distance to table edge
```
Comment (reinzor) [inline robot_smach_states/src/robot_smach_states/util/designators/ed_designators.py]: Hardcoded number ...

**Pull request closed without merging**